sdk/python/foundation-models/system/finetune/image-object-detection/mmdetection-fridgeobjects-object-detection.ipynb

```diff
@@ -449,7 +449,7 @@
    "outputs": [],
    "source": [
     "# Generate jsonl file from coco file\n",
-    "base_url = os.path.join(uri_folder_data_asset.path, \"images\", '')\n",
+    "base_url = uri_folder_data_asset.path + \"/images/\"\n",
```

Comment:
why not to use os.path.join. That would be more safe, right?

Reply:
os.path.join is for local system files. We are editing azureml url, which is more like web-url. The slashes are forward slashes only, in case of web-url.